src/modules/components/timeline/day-item/day-item.scss

```diff
@@ -59,4 +59,10 @@
     color: $secondary-text-colour;
   }
 
+}
+
+gov-day-item:last-child() {
```

Comment:
we can't modify the sass files. We need to use the ones provided by Trevor so the solution needs to find a way to work with those